endpoints/info/bidders.go

```diff
@@ -49,12 +57,27 @@ func NewBidderDetailsEndpoint(infos adapters.BidderInfos, aliases map[string]str
 
 	// Add in any default aliases
 	for aliasName, bidderName := range aliases {
+		// Add the alias bidder info into "allBidderInfo" map
+		aliasInfo := infos[bidderName]
+		aliasInfo.AliasOf = bidderName
+		allBidderInfo[aliasName] = aliasInfo
+
+		// JSON encode core bidder info for the alias and add it to the "responses" map
 		responses[aliasName] = createAliasInfo(responses, aliasName, bidderName)
 	}
 
+	allBidderResponse, err := json.Marshal(allBidderInfo)
+	if err != nil {
+		glog.Fatal("Failed to JSON-marshal all bidder info data.")
+	}
+	// Add the json response containing all bidders info for the /info/bidders/all endpoint
+	responses["all"] = allBidderResponse
```

Comment:
@josephveach good point! I believe we're assuming that `all` will never be a bidder name but I am not so sure about it being an alias name as that can be set in the config by anyone who's hosting PBS (please correct me if I am wrong here, @hhhjort). 

And anyway, I think the most ideal way of doing this would have been to create a totally new endpoint named `info/bidders/all` (since `info/bidders` is already taken to return the list of bidders supported by PBS). I wouldn't mind going that route. What do you guys think?

Reply:
That is the same problem. The current solution returns all bidders on the URL `info/bidders/all`, so you can't define it as a separate endpoint. Might make the name `all-bidders` to reduce the chance of collision. But in any case this should not catch a PBS host by surprise as this should be a "well known" name.

Could perhaps hit a fatal error on startup if the default request defines a conflicting alias.

Reply:
I agree with the fatal error on startup suggestion.  Any reasonable name we choose could lead to a collision.

Reply:
@hhhjort , you're right. I was thinking that if we add it as a separate endpoint and document it then it would be obvious for anyone to not use `all` as an alias name but we are documenting this as an endpoint even now so this should definitely not be a surprise to a PBS host.

But anyway to be on the safer side, I like the idea of a fatal error on startup. Will add that. Thanks @hhhjort and @josephveach !